src/Dotnet.Script.Core/Internal/StringExtensions.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-internal static class StringExtensions
+public static class StringExtensions
```

Comment:
public type in an internal namespace? i'd rather use internals visible to

Reply:
Added `InternalsVisibleTo`